index.html

```diff
@@ -1,21 +1,20 @@
 <!DOCTYPE html>
 
-<html>  
-
-<head>  
-
-    <title> Simple_To_Do_App </title>
-
-</head>
-
-
-<body>   
-
-
-
-
-
-</body>
-
-
-</html>
\ No newline at end of file
+<html>
+  <head>
+    <title>Simple_To_Do_App</title>
+  </head>
+
+  <body>
+    <div class="Container">
```

Comment:
Class names should be lowercase. 
There are some other conventions regarding classnames. The most popular one is 
https://getbem.com/naming/
Please have a look.
```suggestion
    <div class="container">
```